.github/CODEOWNERS

```
@@ -273,7 +273,7 @@
 /Makefile.trace                         @DataDog/agent-apm
 
 /omnibus/                               @DataDog/agent-delivery
-/omnibus/python-scripts/                @DataDog/platform-integrations
+/omnibus/python-scripts/                @DataDog/platform-integrations @DataDog/windows-agent
```

```suggestion
/omnibus/python-scripts/                @DataDog/windows-agent
```
the `platform-integrations` team was split see #33154